### ISSUE TITLE
Card worked briefly then stopped working after computer reset

### ISSUE THREAD

**Issue by Silverfur1** [hanshuebner/AppleII-VGA#18]
Don't know what happened, using Apple IIe running on the PICO ATX Adaptor board. Have flipped the cable, reset board and still black picture. Was going to try to reflash the PICO Pi but unsure as which firmware to use. could you help. Cheers :)

**Comment by hanshuebner** [comment]
I've added firmware images for Pico and Pico W to the repository - Those are the ones that I have used for the cards that I built.  Both require a manual reset using the keyboard after powering up.  Let me know if those work for you.

**Comment by Silverfur1** [comment]
Okay, Plugged the PICO Pie into the USB and RPI came up on my computer briefly then vanished, and the PI is getting red hot, I think it might have a bigger problem :)

**Comment by hanshuebner** [comment]
Sounds like a short to me.

**Comment by Silverfur1** [comment]
yeah, it's only the PI getting hot, not the rest of the board. Would swapping the PI out be an option? (I can work on electronics) :)

**Comment by hanshuebner** [comment]
I'd try whether the pico works without the Apple II card attached.  It could be that the short is on the Apple II card.  Is it the RP2040 that gets hot or is it the voltage regulator?

**Comment by Silverfur1** [comment]
No LED on the PI, and the small components above the bootselect button are the ones getting hot

**Comment by Silverfur1** [comment]
Will desolder the PI tomorrow and see if it works independant, and ordered a new one, only $13 so no biggie :)
